website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -6637,118 +6638,120 @@ allowedClasses:
 
-##### Step 3
+**Step 3**
 
-You can directly use the Java class from your Python pipeline using a stub transform created using the `JavaExternalTransform` API. This API allows you to construct the transform
-using the Java class name and allows you to invoke builder methods to configure the class.
+You can use the Java class directly from your Python pipeline using a stub transform created from the `JavaExternalTransform` API. This API allows you to construct the transform using the Java class name and allows you to invoke builder methods to configure the class.
 
-Constructor and method parameter types are mapped between Python and Java using a Beam Schema. The Schema is auto-generated using the object types
-provided on the Python side. If the Java class constructor method or builder method accepts any complex object types, make sure that the Beam Schema
+Constructor and method parameter types are mapped between Python and Java using a Beam schema. The schema is auto-generated using the object types
+provided on the Python side. If the Java class constructor method or builder method accepts any complex object types, make sure that the Beam schema
 for these objects is registered and available for the Java expansion service. If a schema has not been registered, the Java expansion service will
-try to register a schema using [JavaFieldSchema](https://beam.apache.org/documentation/programming-guide/#creating-schemas). In Python arbitrary objects
-can be represented using `NamedTuple`s which will be represented as Beam Rows in the Schema. See below for a Python stub transform that represents the above
-mentioned Java transform.
+try to register a schema using [JavaFieldSchema](https://beam.apache.org/documentation/programming-guide/#creating-schemas). In Python, arbitrary objects
+can be represented using `NamedTuple`s, which will be represented as Beam rows in the schema. Here is a Python stub transform that represents the above
+mentioned Java transform:
 
-{{< highlight >}}
+```py
 JavaDataGeneratorConfig = typing.NamedTuple(
 'JavaDataGeneratorConfig', [('prefix', str), ('length', int), ('suffix', str)])
 data_config = JavaDataGeneratorConfig(prefix='start', length=20, suffix='end')
 
 java_transform = JavaExternalTransform(
 'my.beam.transforms.JavaDataGenerator', expansion_service='localhost:<port>').create(numpy.int32(100)).withJavaDataGeneratorConfig(data_config)
-{{< /highlight >}}
+```
 
-This transform can be used in a Python pipeline along with other Python transforms.
+You can use this transform in a Python pipeline along with other Python transforms. For a complete example, see [javadatagenerator.py](https://github.com/apache/beam/blob/master/examples/multi-language/python/javadatagenerator.py).
 
-##### 13.1.1.2 Full API for Making Existing Java Transforms Available to Other SDKs
+##### 13.1.1.2 Making existing Java transforms available to other SDKs
```

Comment:
Ditto regarding ambiguity. In both cases (13.1.1.1 and 13.1.1.2) we make existing Java transforms available to other SDKs. The difference is, in 13.1.1.2, users need to write additional Java code just to make such transforms available to other SDKs.